### PR TITLE
Add IPv6 DNS poisioning.

### PR DESCRIPTION
**Pull request by jnc74743**
Add function to define the primary IPv6 address and use that address, addinng it as a host alias for local xrootd gateways.

### CODE REVIEW

**Comment by Jo-stfc** [comment]
lgtm. primary ipv6 are guaranteed to end in : due to the way the ip addresses are assigned

File: docker.py [hunk 1]
```diff
@@ -60,6 +83,11 @@ def args_create(argv):
         dargs.append('--label=xrootd-local-gateway=true')
         dargs.append('--network=ralworker')
         dargs.append('--add-host=xrootd.echo.stfc.ac.uk ceph-gw10.gridpp.rl.ac.uk ceph-gw11.gridpp.rl.ac.uk:172.28.1.1')
+        # Call function to capture primary IPv6 address and assign xrootd alias to local containers IPv6 address.
+        primary_ipv6 = get_primary_ipv6()
+        if primary_ipv6:
+            primary_ipv6 = primary_ipv6.rstrip(':')
+            dargs.append('--add-host=xrootd.echo.stfc.ac.uk ceph-gw10.gridpp.rl.ac.uk ceph-gw11.gridpp.rl.ac.uk:{}{}'.format(primary_ipv6, ':1000:2'))
```
Comment (Jo-stfc) [inline docker.py:90]: could you add svc20 too in this list? it got added as a aaa host recently

Comment (Jo-stfc) [reply docker.py:90]: tbd in a separate commit

Comment (jnc74743) [reply docker.py:90]: We can include this as a separate commit, this should just be a mirror of what we currently do for IPv4, we can diverge at a later time.

**Pull request merged**